dependencies.properties

```diff
@@ -71,7 +71,7 @@ maven.com_google_http_client_google_http_client_gson=com.google.http-client:goog
 maven.org_codehaus_mojo_animal_sniffer_annotations=org.codehaus.mojo:animal-sniffer-annotations:1.18
 maven.javax_annotation_javax_annotation_api=javax.annotation:javax.annotation-api:1.3.2
 maven.org_graalvm_nativeimage_svm=org.graalvm.nativeimage:svm:22.0.0.2
-maven.org_graalvm_sdk=org.graalvm.sdk:graal-sdk:21.3.2
+maven.org_graalvm_sdk=org.graalvm.sdk:graal-sdk:22.1.0
```

Comment:
This is the dependency that was failing Java 8 build https://github.com/googleapis/gax-java/pull/1669

Reply:
Not actionable: Given the new behavior introduced in 22.1.0, this might cause some of our Kokoro GraalVM builds to be temporarily red.

Reply:
I missed to remove this. This was a test to confirm the build passes. Reverted in this PR.

Reply:
Ah got it, thanks.